redbot/cogs/modlog/modlog.py

```diff
@@ -138,8 +144,18 @@ async def casesfor(self, ctx: commands.Context, *, member: Union[discord.Member,
             return await ctx.send(_("That user does not have any cases."))
 
         embed_requested = await ctx.embed_requested()
-
-        rendered_cases = [await case.message_content(embed=embed_requested) for case in cases]
+        if embed_requested:
+            rendered_cases = [await case.message_content(embed=True) for case in cases]
+        elif not embed_requested:
+            rendered_cases = []
+            for case in cases:
+                message = _("{case}\n**Timestamp**: {timestamp}").format(
```

Comment:
The colon is bolder for other fields, so it should be bolded here too for consistency.
```suggestion
                message = _("{case}\n**Timestamp:** {timestamp}").format(
```